Lib/idlelib/debugger.py

```diff
@@ -10,12 +10,20 @@
 
 
 class Idb(bdb.Bdb):
+    """Idle debugger, based on the bdb debugger."""
```

Comment:
I prefer (but don't require) single quotes for single line docstrings, to leave 2 more spaces for a long string,